packages/kotti-ui/source/kotti-field-select/constants.ts

```diff
@@ -31,6 +31,18 @@ export const KOTTI_FIELD_SELECT_SUPPORTS: KottiField.Supports = {
 
 export const KOTTI_FIELD_SINGLE_SELECT_PROPS = {
 	...KOTTI_FIELD_SELECT_PROPS,
+	actions: {
+		default: () => null,
+		type: Array,
+		validator: (value: unknown[]): value is KottiFieldSingleSelect.Action[] =>
+			value.every(
```

Comment:
first, we check that `value` is an `Array` (`Array.isArray`)
then we check that every value matches what it should
b/c u typed it as an `unknown[]` you already assumed, you get some array
how do you guarantee that this is the case